arcaflow_plugin_pcp/pcp_schema.py

```diff
@@ -21,81 +32,28 @@ class PcpInputParams:
             "Timeout in seconds after which to cancel the pmlogger collection"
         ),
     ] = None
-
-
-@dataclass
-class IntervalOutput:
-    interval: typing.Annotated[
-        int,
-        schema.id("@interval"),
```

Comment:
Is it not worth having a fixed output format?

Reply:
I think it's simply not feasible because of the complexity. See my comment about the `any` type -- If I can get at least a little more explicit, that would be great, but I would need some help with that.